src/SprykerSdk/Zed/SprykGui/SprykGuiConfig.php

```diff
@@ -18,14 +17,22 @@ class SprykGuiConfig extends AbstractBundleConfig
      */
     public function getCoreNamespaces(): array
     {
-        return Config::get(KernelConstants::CORE_NAMESPACES, []);
+        return $this->get(KernelConstants::CORE_NAMESPACES, []);
     }
 
     /**
      * @return array
      */
     public function getProjectNamespaces(): array
     {
-        return Config::get(KernelConstants::PROJECT_NAMESPACES, []);
+        return $this->get(KernelConstants::PROJECT_NAMESPACES, []);
+    }
+
+    /**
+     * @return bool
+     */
+    public function isSprykWebInterfaceEnabled(): bool
```

Comment:
IMO this check is not needed as Spryks should only be installed in dev mode and we do not have this kind of checks anywhere else.

Reply:
I agree with you. The requirement for this at the time came from the product side.
I was also always against it :) But now that we have it, it shouldnt hurt to have either.

Reply:
Only in the sake of BC.

Reply:
But at least we are giving the way to override it instead of hardcode.